### ISSUE TITLE
The required [country/city] timezone information is not provided!

### ISSUE THREAD

**Issue by kachihro** [telerik/kendo-react#441]
## I'm submitting a...
Question / Bug ?

* Bug report 

## Current behavior
When setting the timezone, as with the documentation, there is an error shown, I haven't been able to successfully configure TIMEZONE for the React Scheduler component.

## Expected behavior
Format of times to be 'local' - and also DD/MM/YYYY by default - rather than US-only 'MM/DD'

## Minimal reproduction of the problem with instructions

** Within a TSX component ;

const locale = 'en-AU';
const timezone = 'Australia/Sydney';

&lt; IntlProvider locale={locale} &gt; 
                &lt;  Scheduler
                    data = { dataForScheduler() }
                    timezone = { locale }
                    date = { date }
                / &gt; 
/ &gt; 

Have tried the same issue in the StackBlitz component - it has the same error ;
https://stackblitz.com/run/?file=app%2Fmain.jsx

## Environment

Package versions:
+-- @progress/kendo-date-math@1.5.1
+-- @progress/kendo-drawing@1.6.0
+-- @progress/kendo-react-buttons@3.10.0
+-- @progress/kendo-react-dateinputs@3.10.0
+-- @progress/kendo-react-dialogs@3.10.0
+-- @progress/kendo-react-dropdowns@3.10.0
+-- @progress/kendo-react-form@3.10.0
+-- @progress/kendo-react-inputs@3.10.0
+-- @progress/kendo-react-intl@3.10.0
+-- @progress/kendo-react-popup@3.10.0
+-- @progress/kendo-react-scheduler@3.10.0
+-- @types/node@10.17.13
+-- @types/powerapps-component-framework@1.2.1
+-- @types/react@16.9.17
+-- office-ui-fabric-react@7.82.1
+-- pcf-scripts@1.1.6
+-- pcf-start@1.1.6
+-- react@16.12.0
`-- react-dom@16.12.0

**Comment by kspeyanski** [comment]
Hello @kachihro ,

In the provided code snippet i can see you are setting the `timezone` property of the `scheduler` to the local `locale` variable, instead of the `timezone`.

Could you please set the `timezone` to `Australia/Sydney`:
```
<IntlProvider locale={locale}>
<Scheduler
    data = { dataForScheduler() }
    timezone = { timezone }
    date = { date }
/>
/>
```

And for the DateFormat, the corresponding `intl` data must be loaded. I have prepared the following stackblitz example demonstrating how to load intl data for the `en-AU` locale, in order to show the correct dates format in the `Header` and during `form-edit`:
https://stackblitz.com/edit/react-xzq8wf-lnvxyw

For more information about localisation and internationalisation of the `Scheduler`, please refer to the [Scheduler Globalisation](https://www.telerik.com/kendo-react-ui/components/dateinputs/globalization/) article. 

Let me know if you need further assistance.

**Comment by kachihro** [comment]
Thanks - I had actually noticed that - my mistake !    The dates are now showing correctly.

But - changing to SYDNEY doesn't work - as a timezone.    This is also erroring - in the StackBlitz.

`<IntlProvider locale="en-AU" >
                <Scheduler data={sampleData} defaultDate={displayDate} editable={true} timezone = 'Australia/Sydney'>
                    <DayView />
                    <WeekView />
                    <MonthView />
                </Scheduler>
            </IntlProvider>
`

**Comment by kspeyanski** [comment]
I can experience the same error in `stackblitz` too, but in my local workspace, as well as in our [Scheduler Overview](https://www.telerik.com/kendo-react-ui/components/scheduler/#toc-kendoreact-scheduler-demo-preview) changing the timezones does not throw an error.

Can you please elaborate on this one, and confirm that you are experiencing the same error in your local environment? I believe the issue might be with the `stackblitz` example runner.

If the issue still persist in your local environment, is it possible to provide a project where i can reproduce it. This will help me better investigate the issue.

**Comment by kspeyanski** [comment]
We were able to reproduce the issue in a local environment using the `react-script` build script. A hotfix will be published shortly.

**Comment by kspeyanski** [comment]
The issue has been fixed in version `3.13.0` of the `@progress/kendo-react-scheduler` package. After updating all the dependencies, the timezone can be changed successfully in the following stackblitz example:
https://stackblitz.com/edit/react-dwofjh?file=app/main.jsx
